airmozilla/manage/templates/manage/survey_edit_questions.html

```diff
@@ -0,0 +1,82 @@
+{% extends 'manage/surveys.html' %}
+
+{% block manage_title %}
+    Questions for "{{survey.name}}"
+{% endblock %}
+
+{% block site_css %}
+  {{ super() }}
+  {% compress css %}
+  <link href="{{ static('select2/select2.css') }}" rel="stylesheet" type="text/css">
+  <link href="{{ static('select2/select2-bootstrap.css') }}" rel="stylesheet" type="text/css">
+  <link href="{{ static('manage/css/survey.css') }}" rel="stylesheet" type="text/css">
+  {% endcompress %}
+{% endblock %}
+
+{% block site_js %}
+  {{ super() }}
+  <script src="{{ static('select2/select2.min.js') }}"></script>
+  {% compress js %}
+    <script src="{{ static('manage/js/survey-edit.js') }}"></script>
+  {% endcompress %}
+{% endblock %}
+
+{% block content_class %}col-md-6{% endblock %}
+
+{%block manage_content %}
```

Comment:
`{% block manage_content %}` instead of `{%block manage_content %}`.

I know it sounds like an unimportant detail. And it is. And it's not. Quality is important because it makes projects easier to maintain. 
Have you heard of the [Broken Window Theory](https://en.wikipedia.org/wiki/Broken_windows_theory)
Basically, if an abandoned building has broken windows, people think that it's so abandoned that anybody can spray graffiti or demolish it or do whatever they like. Just because it has broken windows. But if the windows are all fixed people leave it alone and don't make it worse. 

The same goes with code. If the code is neat, structured and consistent people who come and work on it will continue to uphold that tradition and will continue to follow that line. 
If the code is sloppy and different in different places it sets your mind that it's not important what you write so the quality slowly drops. 

And consistency is important because it makes it easier to read the code because you're mind doesn't get hung up on odd differences. 

Sorry for the long essay. I hope it helps. I hope the airmozilla code is going to last for a long time and continue to be maintained for many years.